src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -165,11 +167,16 @@ def _extract_message(
         msg: Message[KafkaPayload],
     ) -> ParsedMessage:
         assert isinstance(msg.value, BrokerValue)
+        decoded_str = msg.payload.value.decode("utf-8")
+
         try:
-            parsed_payload: ParsedMessage = json.loads(
-                msg.payload.value.decode("utf-8"), use_rapid_json=True
-            )
-        except rapidjson.JSONDecodeError:
+            if in_random_rollout("sentry-metrics.indexer.enable-orjson"):
+                # Always create a span because json.loads passes skip_trace=False
+                with sentry_sdk.start_span(op="sentry.utils.json.loads"):
+                    parsed_payload: ParsedMessage = orjson.loads(decoded_str)
+            else:
+                parsed_payload: ParsedMessage = json.loads(decoded_str, use_rapid_json=True)
```

Comment:
if we can show they're functionally equivalent we should just kill rapidjson

Reply:
yes, that's the ultimate goal I suppose.

Reply:
For context: we go through this code 250k times a second as the indexer is the highest throughput consumer we have so it would likely highlight the impact. The next test will be on the ingest consumer.

Reply:
are we worried about the perf impact of `start_span` on this?